File: Circle_Area/area.py
# Algorithm for calculate the Area of a circle
import math

def circle_area():
    radius = eval(input("Enter the radius of the circle: ")) # INPUT
    area = math.pi * (radius * radius)  # PROCESS
    return area

# OUTPUT
print("The Area of the cirlce = {}".format(circle_area()))
print("The Area of the cirlce = {}".format(circle_area()))

File: Circle_Area/area.cpp
# include <iostream>
# include <cmath>
using namespace std;

int main(){

    double radius;
    double area;

    // INPUTS
    cout << "Enter the radius of the circle: ";
    cin >> radius;

    // PROCESS
    area = M_PI * (radius * radius);

    // OUTPUT
    cout << "Area = " << area;

    return 0;
}